build/yamls/windows/containerd-with-ovs/conf/Run-AntreaOVS-Containerd.ps1

```diff
@@ -2,11 +2,23 @@ $ErrorActionPreference = "Stop"
 $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
 $mountPath = ($mountPath.Replace('\', '/')).TrimEnd('/')
 $env:PATH = $env:PATH + ";$mountPath/Windows/System32;$mountPath/openvswitch/usr/bin;$mountPath/openvswitch/usr/sbin"
+$OVSDriverDir="$mountPath\openvswitch\driver"
+$WorkDir="$mountPath\openvswitch\"
+
+#Install OVS Driver
+net start vmms
```

Comment:
We should check if the driver has been installed first, if not we will start installing a signed driver in container.